apps/condo/domains/meter/tasks/sendSubmitMeterReadingsPushNotifications.js

```diff
@@ -96,9 +96,17 @@ const readMeterReportingPeriods = async ({ context, organizations }) => {
     })
 }
 
-const checkIsDateStartOrEndOfPeriod = (date, today, start, end) => {
-    return dayjs(date).format('YYYY-MM-DD') === dayjs(today).set('date', start).format('YYYY-MM-DD') ||
-        dayjs(date).format('YYYY-MM-DD') === dayjs(today).set('date', end).format('YYYY-MM-DD')
+const checkIsDateStartOrEndOfPeriod = (date, today, start, end) => (
```

Comment:
```suggestion
const isTodayAtEndOrStartOfPeriod = (date, start, end) => (
```

Reply:
I did this to avoid unnecessary initialization of the `dayjs` object